packages/server/src/db/tests.rs

```diff
@@ -52,3 +52,34 @@ pub async fn test_db_internal_error() {
         );
     }
 }
+
+#[test]
+pub async fn test_get_courses_by_filters() {
+    dotenv().ok();
+
+    let client = init_mongodb_client!();
+
+    let courses = db::services::get_courses_by_name("חשבון אינפיניטסימלי 1מ'", &client)
+        .await
+        .expect("Failed to get courses by name");
+
+    assert_eq!(courses.len(), 1);
+    assert_eq!(courses[0].name, "חשבון אינפיניטסימלי 1מ'");
+    assert_eq!(courses[0].id, "104031");
+
+    let courses = db::services::get_courses_by_number("104031", &client)
+        .await
+        .expect("Failed to get courses by number");
+
+    assert_eq!(courses.len(), 1);
+    assert_eq!(courses[0].name, "חשבון אינפיניטסימלי 1מ'");
+    assert_eq!(courses[0].id, "104031");
+
+    let courses = db::services::get_courses_by_ids(vec!["104031"], &client)
```

Comment:
I know it probably would work.
However, if we test this service, which should be used to get more than one course, let's use more than one course in the vector.